configs/miniTreeEntries.dat

```diff
@@ -1,27 +1,90 @@
 #priorCommitNumberForWhichTheseEventCountNumbersAreValid
-#e0698b415141f838c3b609ae86fbf1306afbcb04
-# productionTag _SHv15noRochester
+#528f48375e743dc42edaaf9792535e59bd531842
+# productionTag _SHv16
 # Dataset miniTree_flavoursideband miniTree_lowdileptonsideband miniTree_signal_ee miniTree_signal_mumu miniTree_dytagandprobe 
-DYJets_amctnlo 196 157499 5172 6964 9996946
-DYJets_madgraph 34 27751 1177 1518 3041984
-DYToEE_powheg_120to200 0 13139 48 0 32418
-DYToEE_powheg_1400to2300 8 10 55947 0 50594
-DYToEE_powheg_200to400 0 195 29142 0 34865
-DYToEE_powheg_2300to3500 7 9 59250 0 47985
-DYToEE_powheg_3500to4500 16 5 60767 0 44948
-DYToEE_powheg_400to800 2 23 41592 0 48318
-DYToEE_powheg_4500to6000 14 10 59381 0 41905
-DYToEE_powheg_50to120 6 8942 3 0 1250963
-DYToEE_powheg_6000toInf 14 3 50407 0 39798
-DYToEE_powheg_800to1400 2 25 50731 0 51549
-DYToEE_powheg 33 208811 22396 0 20598344
-DYToMuMu_powheg_120to200 2 17536 0 137 74462
-DYToMuMu_powheg_1400to2300 10 8 1 59718 95741
-DYToMuMu_powheg_200to400 8 523 0 38538 87023
-DYToMuMu_powheg_2300to3500 6 4 0 58958 95812
-DYToMuMu_powheg_3500to4500 8 3 0 58218 95189
-DYToMuMu_powheg_400to800 13 26 0 56216 94066
-DYToMuMu_powheg_4500to6000 6 8 0 56450 94687
-DYToMuMu_powheg_50to120 4 11714 0 2 1622657
-DYToMuMu_powheg_6000toInf 17 12 2 51856 93227
-DYToMuMu_powheg_800to1400 7 17 0 65799 96024
+# DYJets_amctnlo_SHv16 dataset not found
+DYJets_madgraph 40 661932 1256 1635 3041984
```

Comment:
Signal required !flavoursideband selection string, so I expected to see
more events in signal minitrees.


Regards,
Sean Kalafut

Physics PhD candidate
University of Minnesota Twin Cities

On Thu, May 12, 2016 at 4:56 PM, shervin86 <notifications@github.com> wrote:

> In configs/miniTreeEntries.dat
> <https://github.com/UMN-CMS/cms-WR/pull/50#discussion_r63034925>:
>
> > -DYToEE_powheg_50to120 6 8942 3 0 1250963
> > -DYToEE_powheg_6000toInf 14 3 50407 0 39798
> > -DYToEE_powheg_800to1400 2 25 50731 0 51549
> > -DYToEE_powheg 33 208811 22396 0 20598344
> > -DYToMuMu_powheg_120to200 2 17536 0 137 74462
> > -DYToMuMu_powheg_1400to2300 10 8 1 59718 95741
> > -DYToMuMu_powheg_200to400 8 523 0 38538 87023
> > -DYToMuMu_powheg_2300to3500 6 4 0 58958 95812
> > -DYToMuMu_powheg_3500to4500 8 3 0 58218 95189
> > -DYToMuMu_powheg_400to800 13 26 0 56216 94066
> > -DYToMuMu_powheg_4500to6000 6 8 0 56450 94687
> > -DYToMuMu_powheg_50to120 4 11714 0 2 1622657
> > -DYToMuMu_powheg_6000toInf 17 12 2 51856 93227
> > -DYToMuMu_powheg_800to1400 7 17 0 65799 96024
> > +# DYJets_amctnlo_SHv16 dataset not found
> > +DYJets_madgraph 40 661932 1256 1635 3041984
>
> Number of events increased in sidebands as expected by the bug fix.
> I don't remember if it should have an impact on the signal region as well.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UMN-CMS/cms-WR/pull/50/files/6f6e24e45f157b2a53ac61b7289cc99fe5b1ef40#r63034925>
>